src/main/java/com/google/cloud/genomics/dataflow/pipelines/CalculateCoverage.java

```diff
@@ -131,6 +131,12 @@
     int getNumQuantiles();
 
     void setNumQuantiles(int numQuantiles);
+
+    @Description("The annotation set name")
```

Comment:
Hi Dion,

Looks really nice, but maybe adding a little more description like for the other parameters might help.  Maybe something along these lines;
```Java
@Description("This provides the name for the AnnotationSet.  Default (empty) will proceed to set the "
           + "References.  If References is empty then that will retrieve all the references.  For more "
           + "regarding the definition of an AnnotationSet, please visit:"
           + "https://cloud.google.com/genomics/v1beta2/reference/annotationSets#resource")
```
Hope it helps,
~p